docs/docs/installation.mdx

```diff
@@ -97,7 +97,7 @@ $ENV:BOT_TOKEN="REPLACE_THIS_WITH_YOUR_BOT_TOKEN"
 <TabItem value="linux">
 
 ```bash
-export BOT_TOKEN=<your bot token>
+export BOT_TOKEN="<your bot token>"
```

Comment:
replace with above text

Reply:
Not sure what you want me to do ? Do you want me to replace the text to match the other examples ?